release

```diff
@@ -24,4 +24,3 @@ with open(VERSION_FILE, 'r') as version_file:
 with open(VERSION_FILE, 'w') as version_file:
     for line in lines:
         version_file.write(line.replace(old_version, new_version))
-
```

Comment:
if we're requiring a diff in order to raise a pull request / test a release can we create a new file which we can operate on without needing to come up with creative edits on source files?